pages/game.tsx

```diff
@@ -2,8 +2,15 @@ import Head from "next/head";
 import Display from "../components/display/Display";
 import GamePage from "../components/game/GamePage";
 import styles from "../styles/Home.module.css";
+import React, { useState } from "react";
 
 export default function Home() {
+  const [showHighscore, setShowHighscore] = useState(false);
+
+  setTimeout(() => {
+    setShowHighscore(true);
+  }, 5000);
```

Comment:
This ends in a infinite loop which sets `showHighscore`  every 5000ms.
Make sure to call it only once and not on every render (use `useEffect`)